libtenzir/builtins/contexts/geoip.cpp

```diff
@@ -402,7 +402,11 @@ class ctx final : public virtual context {
                                          "'{}': {}",
                                          db_path_, MMDB_strerror(status)));
     }
-    return show();
+    auto query_f = [](parameter_map) -> caf::expected<expression> {
+      return trivially_true_expression();
+    };
+    return update_result{.update_info = show(),
+                         .make_query = std::move(query_f)};
```

Comment:
```suggestion
    return update_result{.update_info = show(),
                         .make_query = {}};
```

Reply:
Fixed in https://github.com/tenzir/tenzir/pull/3721/commits/c4bc1bdb4ff97c61f7cbc73c34439fefa1081d28